src/main/java/com/fasterxml/jackson/databind/deser/DeserializerCache.java

```diff
@@ -111,6 +120,24 @@ public void flushCachedDeserializers() {
         _cachedDeserializers.clear();
     }
 
+    /**
+     * Constructs and returns {@link LookupCache} instance with default settings.
+     * 
+     * @since 2.16
+     */
+    public static LookupCache<JavaType, JsonDeserializer<Object>> defaultCache() {
```

Comment:
No, I think `DeserializerCache` should not have these -- it should always be provided `LookupCache` from outside. So when constructing initial instance, caller (`DefaultDeserializationContext`) would use `CacheProvider` to create `LookupCache` and pass that.
This can be backwards-incompatible change, I think.